docs/daemon/configuration/configuration.mdx

```diff
@@ -0,0 +1,139 @@
+---
+sidebar_position: 1
+sidebar_label: "Configuration"
+id: daemon-config
+title: Configuration
+description: How to configure the Kwil Daemon
+slug: /daemon/config/
+---
+
+This section outlines the complete set of configurations available for the operation of a kwild node. The configurations are categorized into three distinct types: chain-specific configurations that pertain to the underlying blockchain functionality, app-specific configurations that govern the behavior of kwild application and logging. 
```

Comment:
Personal nit-pick: could we use the Oxford comma?  Especially when saying how long a list is: "chain-specific configurations that pertain to the underlying blockchain functionality, app-specific configurations that govern the behavior of kwild application, and logging."